pmoves/services/model-registry/main.py

```
@@ -0,0 +1,511 @@
+    """Simple Supabase client for model registry operations."""
+
+    def __init__(self, url: str, key: str):
+        self.url = url.rstrip('/')
+        self.key = key
+        self.headers = {
+            "apikey": key,
+            "Authorization": f"Bearer {key}",
+            "Content-Type": "application/json"
+        }
+
+    async def _request(self, method: str, table: str, data: Dict = None, params: Dict = None) -> Dict:
```

Ruff (0.14.13)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)